drivers/sensor/adxl362/Kconfig

```diff
@@ -58,4 +58,56 @@ config ADXL362_ACCEL_ODR_400
 
 endchoice
 
+choice
+	prompt "Trigger mode"
+	default ADXL362_TRIGGER_NONE
+	help
+	  Specify the type of triggering used by the driver.
+
+config ADXL362_TRIGGER_NONE
+	bool "No trigger"
+
+config ADXL362_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+	select ADXL362_TRIGGER
+
+config ADXL362_TRIGGER_OWN_THREAD
+	bool "Use own thread"
+	depends on GPIO
+	select ADXL362_TRIGGER
+
+endchoice
+
+config ADXL362_TRIGGER
+	bool
+
+config ADXL362_THREAD_PRIORITY
+	int "Thread priority"
+	depends on ADXL362_TRIGGER_OWN_THREAD && ADXL362_TRIGGER
+	default 10
+	help
+		Priority of thread used by the driver to handle interrupts.
```

Comment:
Please fix the indentation. It should be two spaces after help

Reply:
Fixed.